gammu/sms_gateway.py

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/env python3
+"""
+SMS Gateway to interact with a GSM modem as a Home Assistant add‐on.
```

Comment:
This should not be part of the add-on itself, but a Python project on it's own. It is easier to have Python specific CI tooling with that approach, and makes this repo minimalistic.